shell/platform/android/image_external_texture_gl.cc

```diff
@@ -36,26 +36,17 @@ ImageExternalTextureGL::ImageExternalTextureGL(
 
 void ImageExternalTextureGL::Attach(PaintContext& context) {
   if (state_ == AttachmentState::kUninitialized) {
-    if (!android_image_.is_null()) {
```

Comment:
This code is invoked by Paint and without this code the first frame after the app resumes will be blank. We have one Image cached and can construct a DlImage here so that it is displayed on the first paint.

Reply:
I'm not sure I'm following here. it looks like all this code is doing is creating a new egl image conditionally, but we should have the egl images cached per dl iamge, so we don't ever rely on reusing one that was already set up?

Reply:
We clear the egl images & textures in Detach so the cache is actually pointing at invalid texture ids and egl images. You need to clear the cache in detach and then here in the case that android_image_ isn't null, create a new eglImage/TextureId/DlImage for the frame held in android_image_.

Reply:
Okay - I see that ProcessFrame essentially handles the case where we get a new image. So if we're re-attaching, then we need to handle setting up the previous image, otherwise we'll just early return there